common/components/controllers/LogInController.jsx

```diff
@@ -24,7 +24,8 @@ class LogInController extends React.Component<Props, State> {
   constructor(props): void {
     super(props);
     const args: Dictionary<string> = url.arguments();
-    const prevPage: string = props.prevPage || args["prev"];
+    let checkPrevPage: string = props.prevPage || args["prev"];
+    const prevPage: string = (checkPrevPage === 'SignUp' ? "Home" : checkPrevPage);
```

Comment:
Use the constants Section.Home and Section.SignUp rather than hard-coding the strings.

Reply:
I have made the changes as requested. Please take a look and let me know.